### PR TITLE
Animate menu icon

### PR DESCRIPTION
**Pull request by dani-moreno**
Transition the color of the icon between states, as well as the menu item does.
Also, uses the correct token for the active icon.

### CODE REVIEW

File: lib/experimental/Navigation/Sidebar/Menu/index.tsx [hunk 1]
```diff
@@ -43,8 +43,8 @@ const MenuItemContent = ({
       <div className="flex items-center gap-1.5 font-medium text-f1-foreground">
         <IconComponent
           className={cn(
-            "h-5 w-5",
-            active ? "text-f1-foreground" : "text-f1-icon"
+            "h-5 w-5 transition-colors",
+            active ? "text-f1-icon-bold" : "text-f1-icon"
```
Comment (josepjaume) [inline lib/experimental/Navigation/Sidebar/Menu/index.tsx:47]: Is it like really fast? Not sure I'm seeing a transition there 👀 .

Also, random thought: Should we eventually add some guidelines on animations? Like, which things we believe are "good" animations and which things aren't?

**Review by josepjaume — APPROVED** [review]

File: lib/experimental/Navigation/Sidebar/Menu/index.tsx (hunk 1 above)
Comment (dani-moreno) [reply lib/experimental/Navigation/Sidebar/Menu/index.tsx:47]: Ah, there seem to be some override with the Navigation Item component and the animation does not happen? It works with a regular element.

Comment (dani-moreno) [reply lib/experimental/Navigation/Sidebar/Menu/index.tsx:47]: Btw, about animation guidelines, I will! It's a bit soon yet but we can have something like [this](https://polaris.shopify.com/design/motion) in the near near future.

Comment (josepjaume) [reply lib/experimental/Navigation/Sidebar/Menu/index.tsx:47]: That would be amazing!

**Pull request merged**